src/routes/(console)/project-[region]-[project]/functions/usage/[[period]]/+page.svelte

```diff
@@ -1,20 +1,86 @@
 <script lang="ts">
     import { base } from '$app/paths';
     import { page } from '$app/state';
+    import { BarChart } from '$lib/charts/index.js';
+    import Card from '$lib/components/card.svelte';
+    import { humanFileSize } from '$lib/helpers/sizeConvertion.js';
     import { Container, Usage } from '$lib/layout';
+    import { total as totalMetrics } from '$lib/layout/usage.svelte';
+    import { IconChartSquareBar } from '@appwrite.io/pink-icons-svelte';
+    import { Icon, Layout, Typography } from '@appwrite.io/pink-svelte';
 
     export let data;
-    $: total = data.deploymentsTotal;
-    $: count = data.deployments;
+    $: total = data.executionsTotal;
+    $: count = data.executions;
+    $: gbHoursTotal = (data.buildsMbSecondsTotal + data.executionsMbSecondsTotal) / 1000 / 3600;
+    $: gbHoursCount = [...data.buildsMbSeconds, ...data.executionsMbSeconds]
+        ?.map((metric) => ({
+            ...metric,
+            value: metric.value / 1000 / 3600
+        }))
+        .filter(({ value }) => value);
+    $: bandwidth = [];
 </script>
 
 <Container>
     <Usage
         path={`${base}/project-${page.params.region}-${page.params.project}/functions/usage`}
         countMetadata={{
-            legend: 'Functions',
-            title: 'Total functions'
+            legend: 'Executions',
+            title: 'executions'
         }}
         {total}
         {count} />
+    
+    <Usage
+        hidePeriodSelect
+        path={`${base}/project-${page.params.region}-${page.params.project}/functions/usage`}
```

Comment:
use a const derived variable and build urls with `resolve` from svelte navigate. type safe and autocompletion. goes for all other files as well.